src/domain/common/whiteboard/excalidraw/ExcalidrawWrapper.tsx

```diff
@@ -74,13 +75,14 @@ const ExcalidrawWrapper = forwardRef<ExcalidrawAPIRefValue | null, WhiteboardWhi
         excalidraw?.zoomToFit();
 
         // Find the properties present in `state.files` and missing in currentFiles
-        // And put them into a BinaryFileData[]
+        // and put them into missingFiles: BinaryFileData[]
         const currentFiles = excalidraw?.getFiles() ?? {};
-        const missingFiles = compact(
-          Object.keys(state.files).map(key => (currentFiles[key] ? undefined : state.files[key]))
+        const newFiles = state.files ?? {};
```

Comment:
Is `?? {}` needed if `RefreshWhiteboardStateParam` now expects to have `files` property always?